etc/mview.xml

```diff
@@ -41,7 +41,7 @@
             <table name="catalog_product_entity_text" entity_column="entity_id" />
             <table name="catalog_product_entity_tier_price" entity_column="entity_id" />
             <table name="catalog_product_entity_varchar" entity_column="entity_id" />
-            <table name="catalog_product_link" entity_column="product_id" />
+            <table name="catalog_product_link" entity_column="product_id" subscription_model="Aligent\PrerenderIo\Model\Mview\View\Attribute\CatalogProductLinkSubscription"/>
```

Comment:
@mausham-shrestha-aligent Please rename the module namespace PrerenderIo to Prerender otherwise we're getting error.

`Warning: class_implements(): Class Aligent\PrerenderIo\Model\Mview\View\Attribute\CatalogProductLinkSubscription does not exist and could not be loaded in /app/vendor/magento/framework/Mview/Config/Converter.php on line 108`